src/ribbon/art_aui.cpp

```diff
@@ -110,16 +110,19 @@ wxColour wxRibbonAUIArtProvider::GetColour(int id) const
     switch(id)
     {
     case wxRIBBON_ART_PAGE_BACKGROUND_COLOUR:
+        wxFALLTHROUGH;
```

Comment:
Do we really need fall through between consecutive cases? I thought none of the compilers warned about this, do they? If not, I'd rather not add this.

Reply:
Yeah, I haven't seen warnings; I sort of assume that future compilers will warn about this as they become more pedantic. But I can certainly roll these back if you prefer.

Reply:
OK, I'm removing those.

Reply:
Sorry, it seems they didn't get removed. I guess it was unintentional/forgotten?

Reply:
OK, fix them.